### ISSUE TITLE
Send DM confirmation when someone disables their account

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#1322]
Ie the inverse of #1024.

**Comment by qazmlp** [comment]
That's probably not going to work (unless a method other than blocking is used to revoke the opt-in).

**Comment by snarfed** [comment]
Definitely, the DM would need to come from a different account, eg @fed.brid.gy@fed.brid.gy.
